Objects/object.c

```diff
@@ -1852,6 +1852,23 @@ PyObject_Not(PyObject *v)
     return res == 0;
 }
 
+/* Test whether two objects are equal */
+
+int PyObject_IsEqual(PyObject *self, PyObject *other)
+{
+    PyObject* res = _Py_BaseObject_RichCompare(self, other, Py_EQ);
+
+    if (res == Py_True)
+        return 1;
+    else if (res == Py_False)
+        return 0;
+    else if (res == Py_NotImplemented)
+        return 0;
+    else
+        return 0;
```

Comment:
Please, follow the PEP 7

Reply:
Happy to do so - is this missing braces? If so I followed the existing convention in this file. Happy to submit a separate PR to change any existing code if that's helpful?

Reply:
Yep, that's about the braces. No, refactoring the rest of the code is not necessary. I'd just prefer to change this new code instead.